packages/angular-output-target/src/output-angular.ts

```diff
@@ -72,16 +72,20 @@ export function generateProxies(
 
   const imports = `/* tslint:disable */
 /* auto-generated angular directive proxies */
-import { ChangeDetectionStrategy, ChangeDetectorRef, Component, ElementRef, NgZone } from '@angular/core';
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, ElementRef, EventEmitter, NgZone } from '@angular/core';
+import { EventEmitter as StencilEventEmitter } from '@stencil/core';
```

Comment:
something like that could work
```ts
 ...
  const publicRuntimeFilePath = path.join(rootDir, distTypesDir, 'stencil-public-runtime');
  const publicRuntimeTypeFile = relativeImport(outputTarget.directivesProxyFile, publicRuntimeFilePath, '.d.ts');

  let imports = `/* tslint:disable */
/* auto-generated angular directive proxies */
import { ChangeDetectionStrategy, ChangeDetectorRef, Component, ElementRef, EventEmitter, NgZone } from '@angular/core';

import { ProxyCmp, proxyOutputs } from './angular-component-lib/utils';\n`;

  if( !outputTarget.componentCorePackage){
    imports += `import { EventEmitter as StencilEventEmitter } from '${normalizePath(publicRuntimeTypeFile)}';`
  } else {
    imports += `import { EventEmitter as StencilEventEmitter } from '${normalizePath(path.join(outputTarget.componentCorePackage, distTypesDir, 'stencil-public-runtime'))}';`
  }
```

Reply:
Sorry for my unresponsiveness, I am on holiday.
Why don't you create pull request based on this one to push it to the finish line? :)